crates/biome_service/src/workspace_watcher.rs

```diff
@@ -113,7 +129,7 @@ impl WorkspaceWatcher {
     /// end of the instructions channel) or the watcher (unexpectedly) stops.
     /// Under normal operation, neither should happen before the daemon
     /// terminates.
-    #[tracing::instrument(level = "debug", skip(self, workspace))]
+    //#[tracing::instrument(level = "debug", skip(self, workspace))]
```

Comment:
Thank you for fixing the issue!
Just a nitpick, should this be reenabled or removed completely? @arendjr

Reply:
Oh, good catch! I'll uncomment that again :)